omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -638,7 +638,7 @@ def draw_rectangle(self, shape):
         p.append(p[0])
         points = p
 
-        stroke_width = scale_to_export_dpi(shape.get('strokeWidth', 2))
+        stroke_width = scale_to_export_dpi(int(shape.get('strokeWidth', 2)))
```

Comment:
It looks like [scale_to_export_dpi](https://github.com/ome/omero-figure/blob/80af88894ddac0f867dc06f827bafe59b13b8aae/omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py#L106) can take floats. Does that lead to weirdness?

Reply:
Good catch @joshmoore. Yes, we do allow floats. The UI allows you to choose `0.25, 0.5, 0.75, 1, 2...`
I see that `scale_to_export_dpi(shape.get('strokeWidth', 2))` occurs in a few other places too, so they'll need the same fix too.
I should probably review `scale_to_export_dpi(pixels)` to do it's own casting of `pixels` to `float` but that might change the return type (int to float) in some cases so I'd want to check the impact of that.